draft-ietf-taps-interface.md

```diff
@@ -727,12 +730,9 @@ have one of five preference levels:
    | Ignore     | No preference                                                          |
    | Avoid      | Prefer protocols/paths not providing the property, proceed otherwise   |
    | Prohibit   | Select only protocols/paths not providing the property, fail otherwise |
+   | Default    | Use the implementation's default preference level for this property    |
```

Comment:
I agree that unsetting a variable could be a good concept, but I'd be afraid of the number of possible changes to the whole document that this approach might incur... is this worth it?

Anyway, either way is fine with me; even just keeping "Default" as a preference level, I could live with, it's not a big deal IMO.

Reply:
I don't think we have many places where this would require a change – just in the examples and the description of Preconnection / TransportProperties. Default values stay default values, unset/reset just restores it.

Reply:
When would you want to "reset" anything?

Reply:
Just agree/do anything ... this isn't super important!